app/controllers/users_controller.rb

```diff
@@ -5,14 +5,49 @@ class UsersController < ApplicationController
   layout 'main_layout'
 
   def index
-    @gds = current_user.permissions.team_management
+    @user = current_user
```

Comment:
`@user` is not used I believe? The rebase brought it back

Reply:
It is used.